internal/rcm/api_test.go

```diff
@@ -68,61 +66,4 @@ func TestBasicRcmAPI(t *testing.T) {
 			t.Errorf("The response to %s request to %s should match this regex %s but returns %s", c.Method, c.Path, c.ExpectedBodyRegex, response_body)
 		}
 	}
-}
-
-func TestSubmitCompose(t *testing.T) {
```

Comment:
If `FetchPackageList` is used instead of `FetchChecksum`, then we can keep this test due to the mocking support in `FetchPackageList`.

Also if we were to delete this test, I would prefer doing it in the same commit which adds the dnf dependency. However, mocking is the best way I think.

Reply:
The test was reintroduced.